tequilapi/client/http_client.go

```diff
@@ -39,26 +40,27 @@ type httpRequestInterface interface {
 	Do(req *http.Request) (*http.Response, error)
 }
 
-func newHttpClient(baseUrl string, logPrefix string, ua string) *httpClient {
+func newHTTPClient(baseURL string, logPrefix string, ua string) *httpClient {
 	return &httpClient{
 		http: &http.Client{
 			Transport: &http.Transport{},
+			Timeout:   time.Second(120),
```

Comment:
```suggestion
			Timeout:   time.Second*120,
```

Reply:
Maybe even better 120 * time.Second. Looks closer to natural human language (120 seconds)

Reply:
Changed it in suggestion, to be able to apply in one click (if it really works in the GitHub)